post-processor.go

```diff
@@ -288,6 +289,7 @@ func (p *PostProcessor) uploadBox(box, boxPath string) error {
 		Bucket:       aws.String(p.config.Bucket),
 		Key:          aws.String(boxPath),
 		ACL:          aws.String(p.config.ACL),
+		ServerSideEncryption:          aws.String(p.config.ServerSideEncryption),
```

Comment:
What are the possible values for this field? Can you document them in the README?